editor/src/messages/tool/common_functionality/shape_editor.rs

```diff
@@ -259,6 +259,73 @@ impl ShapeEditor {
 		result
 	}
 
+	/// Find the path segement we have clicked upon
+	///
+	/// Returns the [`BezierId`] and t as an f64
+	fn closest_segement(&self, document: &Document, layer_path: &[LayerId], pos: glam::DVec2, tolerance: f64) -> Option<(BezierId, f64)> {
```

Comment:
s**ege**ment is used in here and in comments in `graphene/src/layers/vector/subpath.rs` is this a typo for segment, or does it mean something special?

Reply:
Just a typo